singularity/Singularity

```diff
@@ -60,7 +60,7 @@ gpgkey=https://www.mongodb.org/static/pgp/server-4.4.asc" >> /etc/yum.repos.d/mo
     ## clone the DPdash digital phenotyping dashboard
     git clone https://github.com/PREDICT-DPACC/dpdash.git
     cd dpdash
-
+    mkdir app_build
```

Comment:
Okay so why don't we `mkdir public` instead of having to change all following?

Reply:
The project has a public directory where the index.html file lives, this directory is where the javascript lives but it gets created when the application is built by webpack.

Reply:
Does not make sense to me. We have been building DPdash forever with this `public/` directory in place. We should not have to rename that to `app_build/` today.